specification/proto-design-goals.md

```diff
@@ -1,19 +0,0 @@
-# Design Goals for OpenTelemetry Wire Protocol
```

Comment:
This was already moved to RFC repo

Reply:
Can we not do this in the current PR and open a different one with this? I want @tigrannajaryan to comment and others.